tests/wast_testsuites.rs

```diff
@@ -16,12 +16,29 @@ fn run_wast(wast: &str, strategy: Strategy) -> anyhow::Result<()> {
     // by reference types.
     let reftypes = simd || wast.iter().any(|s| s == "reference-types");
 
+    // Reference types assumes support for bulk memory.
+    let bulk_mem = reftypes
+        || wast.iter().any(|s| s == "bulk-memory-operations")
+        || wast.iter().any(|s| s == "table_copy.wast")
+        || wast.iter().any(|s| s == "elem_drop.wast")
+        || wast.iter().any(|s| s == "elem-ref-null.wast")
+        || wast.iter().any(|s| s == "memory-copy.wast")
+        || wast.iter().any(|s| s == "imported-memory-copy.wast")
+        || wast
+            .iter()
+            .any(|s| s == "table_copy_on_imported_tables.wast");
+
+    // And bulk memory also assumes support for reference types (e.g. multiple
+    // tables).
+    let reftypes = reftypes || bulk_mem;
```

Comment:
These new conditions look a bit odd to me. It looks like `bulk_mem` is on if `reftypes` is enabled, but the `bulk_mem` also then enables `reftypes` later on. I thought the dependency was that `reftypes` implied `bulk_mem`, right? If so that's already done in `Config` I believe, so we just need to know here if bulk memory is enabled.

Oh actually I think to get the right behavior from `Config` you need to do the bulk memory setter first, and then the reference types setter (it's a bit wonky).

Also, given that we have a folder for bulk memory, could all our local `misc_testsuite` tests related to bulk memory be placed in a same-name folder to avoid having to list all tests here?

Reply:
> Also, given that we have a folder for bulk memory, could all our local `misc_testsuite` tests related to bulk memory be placed in a same-name folder to avoid having to list all tests here?

FWIW, we don't have a bulk memory directory in our misc test suite but I can make one.